### PR TITLE
Add influxdb-ui service

### PR DESCRIPTION
**Pull request by vishh**
Fixed Readme and the kube.sh script.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,6 +45,11 @@ deploy/kube.sh restart
 
 #####Hints
 * To enable memory and swap accounting on the minions follow the instructions [here](https://docs.docker.com/installation/ubuntulinux/#memory-and-swap-accounting)
+* If the Grafana UI is not accessible via the proxy on the master (URL mentioned above), open up port 80 on the nodes, find out the node on which grafana container is running and access Grafana UI @ http://<minion-ip>/
```
Comment (vmarmol) [inline README.md]: nit: s/grafana/Grafana/

**Comment by rjnagal** [comment]
LGTM.

File: README.md (hunk 1 above)
Comment (vishh) [inline README.md]: Done.

**Comment by vishh** [comment]
PTAL @vmarmol

**Comment by vmarmol** [comment]
LGTM

**Pull request merged**